XpringKit/PayID/Generated/APIHelper.swift

```diff
@@ -0,0 +1,63 @@
+// APIHelper.swift
+//
+// Generated by swagger-codegen
+// https://github.com/swagger-api/swagger-codegen
+//
+
+import Foundation
```

Comment:
Do we need to check in generated code?

Reply:
This is a requirement of Carthage, unfortunately. The build system checks out a Github project and builds it. 

If we generate on the fly, then users of this library need all the generation tooling (swagger, protobuf, node, etc) on their local machine, which isn't always true. The errors for this aren't always intuitive and we had a few users early on get stuck.